plone/namedfile/scaling.py

```diff
@@ -20,6 +19,7 @@
 from plone.rfc822.interfaces import IPrimaryFieldInfo
 from plone.scale.interfaces import IImageScaleFactory
 from plone.scale.interfaces import IScaledImageQuality
+from plone.scale.scale import scale_svg_image
```

Comment:
Does this need to specify a minimum version of plone.scale in setup.py?

Reply:
yes, this is probably not a bad idea.

Reply:
I added the minimum plone.scale version in PR #183.